pkg/agent/cache/manager.go

```diff
@@ -148,10 +149,13 @@ type cacheManager struct {
 	isCacheUpdated          bool
 	isPersistedCacheEnabled bool
 	migrators               []cacheMigrate
+	teamRefreshHandler      teamRefreshHandler
 }
 
+type cacheManagerOptions func(*cacheManager)
+
 // NewAgentCacheManager - Create a new agent cache manager
-func NewAgentCacheManager(cfg config.CentralConfig, persistCacheEnabled bool) Manager {
+func NewAgentCacheManager(cfg config.CentralConfig, persistCacheEnabled bool, opt ...cacheManagerOptions) Manager {
```

Comment:
We can remove this, handler is no longer needed

Reply:
ah, yes...  thanks!